homeassistant/components/nmbs/sensor.py

```diff
@@ -431,9 +427,9 @@ def update(self) -> None:
             return
 
         duration = get_ride_duration(
```

Comment:
Any reason we don't use the provided duration in the connection details? https://github.com/tjorim/pyrail/blob/main/pyrail/models.py#L305

Reply:
I should check this one if the number reports the same.

Reply:
The get_ride_duration can be made more easy thanks to the duration.